README.md

```diff
@@ -1,6 +1,9 @@
 # 🏡 Virtual Senior Center
 
-A project to create a virtual community for LGBT seniors in the Massachusetts area.
+The mission of the outstandinglife.org is to improve the quality of life of LGBTQ older adults in Massachusetts. Through this project, C4C aims to help LBFE find new creative ways to connect older adults: engaging hearts, transforming lives and breaking down boundaries. 
+
+(To be updated when website is deployed)
+Website: [Outstanding Life](https://outstandinglife.org/)
 
 ## 👷 Todo
```

Comment:
Maybe we should delete the ##Todo section too?

Reply:
sgtm, removing - can we take note somewhere else unless that URL is final?